packages/wrangler/src/type-generation/index.ts

```diff
@@ -344,7 +344,11 @@ async function generateTypes(
 
 	if (configToDTS.unsafe?.bindings) {
 		for (const unsafe of configToDTS.unsafe.bindings) {
-			envTypeStructure.push(constructType(unsafe.name, "any"));
+			if (unsafe.type === "ratelimit") {
+				envTypeStructure.push(constructType(unsafe.name, "RateLimit"));
+			} else {
+				envTypeStructure.push(constructType(unsafe.name, "any"));
```

Comment:
I'm not sure this is a great idea to fallback to any for other unsafe bindings - people may be defining the unsafe binding types manually which this will conflict with

Reply:
I might be missing something, but isn't this the current behaviour?
This PR is supposed to fix this problem only for the rate limiter unsafe binding

Reply:
You're right - I shouldn't be reviewing code late at night 😅